README.md

```diff
@@ -0,0 +1,29 @@
+# SciGuides
+
+Welcome to SciGuides, a repository made by FRC Team 1155 and 2265 to gather all of our experience and knowledge for public and internal use.
```

Comment:
i feel like it doesn't really make sense to say made by both our teams? Since it's mostly based on the way that we do things. I think it's a little more descriptive to say that it has contributions from 2265